Client/Frontend/Browser/Favicons/FaviconFetcher.swift

```diff
@@ -292,6 +292,17 @@ public class FaviconFetcher {
           if !cache.isCached(url) {
             completion(self.url, self.monogramFavicon)
             return
+          } else {
+            cache.getCachedImage(for: url) { image in
```

Comment:
Technically this doesn't have to be inside an else clause since the if above returns early